test/nitro/claim-sol.test.ts

```diff
@@ -19,9 +19,9 @@ describe("claim (solidity)", function () {
   });
 
   const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
-  const A_ADDRESS = "0x96f7123E3A80C9813eF50213ADEd0e4511CB820f";
-  const B_ADDRESS = "0x53484E75151D07FfD885159d4CF014B874cd2810";
-  const TARGET_CHANNEL_ADDRESS = "0x080678731247781ff0d57c649b6d0ad1a0620df0"; // At some point in the full claim operation, the outcome of this channel must be read and checked
+  const A_ADDRESS = "0x00000000000000000000000096f7123E3A80C9813eF50213ADEd0e4511CB820f".toLowerCase();
+  const B_ADDRESS = "0x00000000000000000000000053484E75151D07FfD885159d4CF014B874cd2810".toLowerCase();
+  const TARGET_CHANNEL_ADDRESS = "0x000000000000000000000000080678731247781ff0d57c649b6d0ad1a0620df0".toLowerCase(); // At some point in the full claim operation, the outcome of this channel must be read and checked
```

Comment:
Do we need the `.toLowerCase()` here? I think we shouldn't need to worry about casing, except for when comparing strings in javascript.

Reply:
Also should `A_ADDRESS` => `A_DESTINATION` etc?

Reply:
> Re: `A_ADDRESS` => `A_DESTINATION`, I've refactored here to declare A and B simlarly to to the other files.
 
It's just that the variable is *not* an ethereum address anymore, since it is not 20 bytes. So shouldn't we call it something else (i.e. a destination)?

> Re: `toLowerCase`, the solidity `claim` function is returning destinations that have been lowercased.

I'm nitpicking really: it seems that the right place to put the `.toLowerCase()` is in the test assertion itself. We ought to be able to feed any casing into the function and for that to be considered valid (and for the test to pass). Given the way we are currently doing the assertion / comparison, however, my suggestion isn't that straightforward. (One option might be to `JSON.stringify(result).toLowerCase()` and compare that.

Reply:
First point: sorry, the change I mentioned was still only local. Incoming...

Second: agreed that we should be able to pass any valid data. I've been trying to figure out where and when the existing workflow breaks.